### ISSUE TITLE
Error when browser language is not set to English

### ISSUE THREAD

**Issue by Webgardener** [opengovfoundation/madison#1060]
Hello,

When using a browser (let's say Firefox) with language set up to French, the angular app won't work. Here is the error displayed on console :

```
Error: Plural Function not found for locale: fr
MessageFormat@http://www.site.fr/vendor/messageformat/messageformat.js:26:1
b/e.setLocale@http://www.site.fr/lib/angular-translate-interpolation-messageformat/angular-translate-interpolation-messageformat.min.js:6:365
c/this.$get</V@http://www.site.fr/lib/angular-translate/dist/angular-translate.min.js:6:9156
p@http://www.site.fr/lib/angular/angular.min.js:7:353
c/this.$get<@http://www.site.fr/lib/angular-translate/dist/angular-translate.min.js:6:9253
e@http://www.site.fr/lib/angular/angular.min.js:40:17
db/z.$injector<@http://www.site.fr/lib/angular/angular.min.js:41:341
d@http://www.site.fr/lib/angular/angular.min.js:39:220
e@http://www.site.fr/lib/angular/angular.min.js:39:490
db/<@http://www.site.fr/lib/angular/angular.min.js:41:388
p@http://www.site.fr/lib/angular/angular.min.js:7:353
db@http://www.site.fr/lib/angular/angular.min.js:41:366
yc/c@http://www.site.fr/lib/angular/angular.min.js:19:463
yc@http://www.site.fr/lib/angular/angular.min.js:20:274
@http://www.site.fr/js/app.js:253:7
f/<@http://www.site.fr/lib/angular/angular.min.js:121:20
sf/this.$get</n.prototype.$eval@http://www.site.fr/lib/angular/angular.min.js:135:215
sf/this.$get</n.prototype.$digest@http://www.site.fr/lib/angular/angular.min.js:132:298
sf/this.$get</n.prototype.$apply@http://www.site.fr/lib/angular/angular.min.js:135:511
g@http://www.site.fr/lib/angular/angular.min.js:88:426
A@http://www.site.fr/lib/angular/angular.min.js:93:35
bg/</w.onload@http://www.site.fr/lib/angular/angular.min.js:94:76
```

**Comment by sethetter** [comment]
@Webgardener Unfortunately we don't currently have French translations for the project. However, I would image it should have defaulted back to English.

Thanks for submitting the issue! We'll look into it.

**Comment by sethetter** [comment]
@Webgardener Fixed in bb18819!
